src/index.ts

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2011-2012 Sam Elsamman
+ Permission is hereby granted, free of charge, to any person obtaining
+ a copy of this software and associated documentation files (the
+ "Software"), to deal in the Software without restriction, including
+ without limitation the rights to use, copy, modify, merge, publish,
+ distribute, sublicense, and/or sell copies of the Software, and to
+ permit persons to whom the Software is furnished to do so, subject to
+ the following conditions:
+
+ The above copyright notice and this permission notice shall be
+ included in all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+/*
+ * ObjectTemplate - n Type System with Benefits
+ */
+
+import {ObjectTemplate} from './ObjectTemplate';
+
+ObjectTemplate.init();
+export default ObjectTemplate;
+
+let amorphicStatic = ObjectTemplate.amorphicStatic;
+let SupertypeSession = ObjectTemplate.amorphicStatic;
```

Comment:
do you know what this is used for?

Reply:
SupertypeSession is used or at least referenced in the other amorphic components (amorphic and persistor). I need to export this for Remoteable.ts and Persistable.ts. I might clean that up later when I clean up Amorphic and Persistor, but at the moment need to export it.